contentcuration/contentcuration/frontend/channelEdit/views/TreeView/index.vue

```diff
@@ -128,6 +128,13 @@
             />
           </div>
         </template>
+        <template #pagination>
+          <div class="pagination-container">
+            <KButton v-if="more" :disabled="moreLoading" @click="loadMore">
+              {{ 'Show more' }}
```

Comment:
String on the lose

Reply:
I had a dream that this was the case - it seems my dreams came true!